charts/values.yaml

```diff
@@ -0,0 +1,5 @@
+image:
+  hub: intel
+  name: trusted-certificate-issuer
+  tag: "latest"
```

Comment:
Is "latest" a good default value? Maybe it is, or maybe we should update the chart to point to the latest release every time we tag a release.

Reply:
Any suggestion are welcome. Maybe on main branch it should be latest?

Reply:
Till we publish the version-tagged images let's live with the 'latest' tag.